parlai/core/torch_ranker_agent.py

```diff
@@ -137,10 +137,6 @@ def get_batch_train_metrics(self, scores):
         targets = torch.arange(batchsize, out=targets)
         nb_ok = (scores.max(dim=1)[1] == targets).float().sum().item()
         self.metrics['train_accuracy'] += nb_ok
-        # calculate mean rank
-        above_dot_prods = scores - scores.diag().view(-1, 1)
-        rank = (above_dot_prods > 0).float().sum().item()
-        self.metrics['rank'] += rank
```

Comment:
i'm confused -- why is this deleted?

Reply:
nevermind i see -- you can add this code back. the bug is fixed by #1559

Reply:
ok